### PR TITLE
Update to 2018.2 SDK

### PR DESCRIPTION
**Pull request by matkoch**
Now targets net461.

### CODE REVIEW

File: src/Arbel.ReSharper.ConfigureAwaitPlugin.Tests/Arbel.ReSharper.ConfigureAwaitPlugin.Tests.csproj [hunk 1]
```diff
@@ -1,10 +1,10 @@
 <?xml version="1.0" encoding="utf-8"?>
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>net45</TargetFramework>
+    <TargetFramework>net461</TargetFramework>
   </PropertyGroup>
   <ItemGroup>
-    <PackageReference Include="JetBrains.ReSharper.SDK.Tests" Version="2018.1.2" />
+    <PackageReference Include="JetBrains.ReSharper.SDK.Tests" Version="2018.2.0-*" />
```
Comment (aelij) [inline src/Arbel.ReSharper.ConfigureAwaitPlugin.Tests/Arbel.ReSharper.ConfigureAwaitPlugin.Tests.csproj]: I'd rather the version be explicit (no `*`)

**Comment by aelij** [comment]
You also need to update the value of the Wave dependency and the package version & description in the nuspec file

**Comment by matkoch** [comment]
Updated. This was mainly to communicate the changes in target framework and wave version.

**Comment by stoberman37** [comment]
Any updates on this PR?  We are upgrading Resharper, and we need this extension.

**Comment by aelij** [comment]
Closing in favor of #26

**Comment by aelij** [comment]
Thanks, I've published a new version

**Pull request closed without merging**